src/Controller/WidgetAllProjectsController.php

```diff
@@ -149,4 +151,36 @@ public function update($projectId)
 
         return $response;
     }
+
+    public function webNotificationUpdate($projectId)
```

Comment:
May I know your suggestion?

Reply:
The best way is create new NotificationController for this feature.

Reply:
Okay, will do.
But I think this will create similar calls with very subtle differences. I only created a JSON format similar to RoR, since we don't have one. See html, json and xml format http://guides.rubyonrails.org/action_controller_overview.html#rendering-xml-and-json-data.